subgraphs/uniswap-forks/protocols/pangolin/src/common/constants.ts

```diff
@@ -0,0 +1,10 @@
+import { BigInt } from "@graphprotocol/graph-ts";
+
+////////////////////
+//////Versions//////
+////////////////////
+
+export const PROTOCOL_SUBGRAPH_VERSION = "1.1.8";
```

Comment:
You can set the subgraph version to 1.0.0

Reply:
Thanks will do this before deploying final subgraph 👍